website/releases/release-0.13.1.md

```diff
@@ -0,0 +1,569 @@
+---
+title: "Release 0.13.1"
+sidebar_position: 2
+layout: releases
+toc: true
+last_modified_at: 2023-05-25T13:00:00-08:00
+---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# [Release 0.13.1](https://github.com/apache/hudi/releases/tag/release-0.13.1) ([docs](/docs/0.13.1/quick-start-guide))
+## Migration Guide
+
+* This release (0.13.1) does not introduce any new table version, thus no migration is needed if you are on 0.13.0.
+* If migrating from an older release, please check the migration guide from the previous release notes, specifically
+  the upgrade instructions in [0.6.0](/releases/older-releases#release-060-docs),
+  [0.9.0](/releases/older-releases#release-090-docs), [0.10.0](/releases/older-releases#release-0100-docs),
+  [0.11.0](/releases/older-releases#release-0110-docs), and [0.12.0](/releases/older-releases#release-0120-docs).
+
+### Bug fixes
+
+0.13.1 release is mainly intended for bug fixes and stability. The fixes span across many components, including
+
+* DeltaStreamer
+* Metadata table and timeline server out of sync issue
+* Table services
+* Spark SQL
+* Presto stability/pref fixes
+* Trino stability/perf fixes
+* Meta Syncs
+* Flink engine
+* Unit, functional, integration tests and CI
+
+## Raw Release Notes
+
+The raw release notes are available [here](https://issues.apache.org/jira/secure/ReleaseNote.jspa?projectId=12322822&version=12352250)
+:::tip
+0.13.1 release also contains all the new features and bug fixes from 0.13,0, of which the release notes are
+shown below.
+:::
+
+# [Release 0.13.0](https://github.com/apache/hudi/releases/tag/release-0.13.0) ([docs](/docs/quick-start-guide))
+
+Apache Hudi 0.13.0 release introduces a number of new features including [Metaserver](#metaserver),
+[Change Data Capture](#change-data-capture), [new Record Merge API](#optimizing-record-payload-handling),
+[new sources for Deltastreamer](#new-source-support-in-deltastreamer) and more.  While there is no table version upgrade
```

Comment:
Why the release note looks same with 0.13.0 ?

Reply:
Hi @danny0405 
These are quote from 0.13.0 after
```

## Raw Release Notes

The raw release notes are available [here](https://issues.apache.org/jira/secure/ReleaseNote.jspa?projectId=12322822&version=12352250)
:::tip
0.13.1 release also contains all the new features and bug fixes from 0.13,0, of which the release notes are
shown below.
:::

# [Release 0.13.0](https://github.com/apache/hudi/releases/tag/release-0.13.0) ([docs](/docs/quick-start-guide))
```

Reply:
Also still work on failed CI case.

Reply:
We only need to keep the changes in 0.13.1 I think.

Reply:
It's fine to keep all 0.13.x docs here, to be consistent with other release notes (see 0.11.1 and 0.12.3).  Let's remove `release-0.13.0.md` since it's deprecated and move the content to `older-releases.md`.